testsuite/sclang/lpc/LPCTestUtils.sc

```diff
@@ -13,60 +13,60 @@
 * fast for SC. You are welcome to find faster methods.
 */
 
-/****** LAYOUT OF THIS FILE *******
-
-----major methods----
-
-evaluateAllStrings                 // main method: take an alphabet, evaluate all
-                                   //   strings in its language, and write to a file.
-evaluateString                     // evaluate a string; called by evaluateAllStrings
-compareFiles                       // compare two files (typically test/validation)
-compareHeaders                     // compare the headers; called by compareFiles
-compareData                        // compare the data; called by compareFiles
-
-----diff helper methods----
-
-writeDiffs                         // write an array of diffs to a file
-explainDiff                        // convert a diff to easily readable format
-makeDiff                           // construct a diff from two dictionaries
-
-----misc helper methods----
-
-makeTestString                     // make a test string given an alphabet and counter
-safeOpenFile                       // open a file, throwing errors immediately on fail
-safeMkdir                          // mkdir if no dir exists, throw error on fail
-parseTestResult                    // turn a line read from a file into structured data
-doOutputsMatch                     // return true iff two test results are considered
-                                   //   matching. see implementation for special cases.
-doFloatOutputsMatch                // return true if floats match for a more accepting
-                                   //   cross-platform interpretation
-incrementAlphabetCount             // increment an alphabet counter in-place
-debug                              // ordinary debugging method
-
-----data conversion methods----
-
-bytecodeToHexString                // Int8Array -> String of hex characters
-bytecodeFromHexString              // inverse of `bytecodeToHexString`
-stringToHexString                  // String of chars -> String of hex chars
-stringFromHexString                // inverse of `stringToHexString`
-
-----header I/O methods----
-
-writeHeader                        // write a header with test parameter info
-parseHeader                        // recover test data from a file's header
-
-----header parsing helper methods----
-
-verifyFieldName                    // these methods all perform specialized minor data
-parseBlockName                     //   validation. see implementations for details.
-parseAlphabetSize
-parseAlphabet
-parseStringLength
-parsePrefix
-parseSuffix
-parseTechnique
-
-************************************/
+// ****** LAYOUT OF THIS FILE *******
```

Comment:
out of curiosity: what is the reason for this conversion? I always found it much easier to deal with `/* – */` than with multiline `//`, and it looks better to me.